### DISCUSSION TITLE
tran auth command

### DISCUSSION THREAD

**Discussion by david-tomson** [abdfnx/tran#8] [Ideas]
Hi @abdfnx, nice job, so I think that we've a config file, and I want to sync it, so I tran has `auth` command, the user can sync his config file anywhere, what do you think @abdfnx ?

**Comment by abdfnx** [comment]
Nice idea @david-tomson, this mission is for you, fork this project, and create a pull request with the new changes

**Comment by abdfnx** [comment]
there's some issues about `sync` command, I'm going to fix it

**Comment by abdfnx** [comment]
Done
